packages/teams-js/src/internal/telemetry.ts

```diff
@@ -12,7 +13,59 @@ registerLogger.formatArgs = function (args) {
   originalFormatArgsFunction.call(this, args);
 };
 
-const topLevelLogger = registerLogger('teamsJs');
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const createDebuggerFunction = (namespace: string): Debugger => {
```

Comment:
This is cool and it will probably need a good dose of explanatory comments

Reply:
Could you please take a look on comments? Basically I just explained the code in detail.